go.mod

```diff
@@ -159,7 +159,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20230109183929-3758b55a6596 // indirect
 	k8s.io/kubectl v0.26.0 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
-	sigs.k8s.io/kind v0.18.0 // indirect
+	sigs.k8s.io/kind v0.20.0 // indirect
```

Comment:
@richardcase I guess the problem is, this will always point to v0.20.0 as its indirect import from CAPI 1.4.4

Reply:
I have made the changes to replace this I think now the tests should pass.

Reply:
Working now 🙂

Reply:
Thanks @damdo for pointing to the right cause 🎉

Reply:
Yes, that's why Richard did the pinning in the root go.mod in the first place, I think.
I saw you now did it as well, so let's see if the e2e pass this time :)

Reply:
:crossed_fingers: